ccpp/suites/suite_FV3_GFS_v15plusras.xml

```diff
@@ -65,7 +65,6 @@
       <scheme>cires_ugwp</scheme>
       <scheme>cires_ugwp_post</scheme>
       <scheme>GFS_GWD_generic_post</scheme>
-      <scheme>rayleigh_damp</scheme>
```

Comment:
May I ask why the Rayleigh damping is removed? Also I'd like to let you know that we are going to remove the old GFDL MP GFSv15 suite files as they are not used by opn or future implementation.

Reply:
@SMoorthi-emc probably answered already, see also https://github.com/NOAA-EMC/fv3atm/issues/340.